terraform/workspace_variables/qa.tfvars

```diff
@@ -0,0 +1,13 @@
+# PaaS
+paas_sso_code                      = ""
+paas_app_environment               = "qa"
+paas_rails_env                     = "qa"
+paas_space_name                    = "bat-qa"
+paas_app_start_timeout             = "180"
+paas_web_app_deployment_strategy   = "blue-green-v2"
```

Comment:
We will always use blue-green-v2 for the webapp so I suggest to remove the variable and hardcode  blue-green-v2  in the module